tmt/steps/prepare/install.py

```diff
@@ -419,7 +419,7 @@ def prepare_command(self) -> Tuple[Command, Command]:
 
         command += Command('rpm-ostree')
 
-        options = Command('--apply-live', '--idempotent', '--allow-inactive')
+        options = Command('--apply-live', '--idempotent', '--allow-inactive', '--assumeyes')
```

Comment:
Would it make sense to make this option conditional? I suppose it's not supported by older (whatever that may mean) `rpm-ostree` versions, adding it to the command line could then cause errors. Is that a valid concern?

Reply:
ahhh yep, failed to consider it. all though, the nature of rpm-ostree is that all images are compiled, I think in theory all tested images should be latest builds? I could be off on this logic though

Reply:
How about testing older (say f35) fedora-coreos images? I don't think we mention miminal possible SUT version.